qt/scientific_interfaces/Indirect/IndirectDiffractionReduction.cpp

```diff
@@ -224,6 +244,10 @@ void IndirectDiffractionReduction::plotResults() {
         showInformationBox(QString::fromStdString(
             "Workspace '" + it + "' not found\nUnable to plot workspace"));
     }
+#else
+    using MantidQt::Widgets::MplCpp::pcolormesh;
+    pcolormesh(toQStringList(m_plotWorkspaces));
+#endif
   }
 
   runPythonCode(pyInput);
```

Comment:
You don't need this anymore, right? at least outside of a `#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)`